test/model/content_test.dart

```diff
@@ -141,10 +141,20 @@ void main() {
     // TODO test group mentions and wildcard mentions
   });
 
-  testParseInline('parse Unicode emoji',
+  testParseInline('parse Unicode emoji, encoded in span element',
     // ":thumbs_up:"
     '<p><span aria-label="thumbs up" class="emoji emoji-1f44d" role="img" title="thumbs up">:thumbs_up:</span></p>',
-    const UnicodeEmojiNode(text: ':thumbs_up:'));
+    const UnicodeEmojiNode(emojiName: ':thumbs_up:', codepoints: '\u{1f44d}'));
```

Comment:
Hmm, I guess this example shows that the thing labeled `emojiName` isn't exactly the name — it's the name (which here is `thumbs_up`) plus colons to decorate it.

One good name would be just `colonsName`.

Reply:
Or perhaps better yet, drop that field entirely, since we're no longer using it. (Easy to add it back later if we start using it again for something.)

That will mean squashing the commit that changes how `widgets/content.dart` consumes UnicodeEmojiNode into the commit that changes what those nodes contain and how we parse them. I think squashing those is probably cleanest anyway.